packages/grid/frontend/src/lib/components/DynamicModal.svelte

```diff
@@ -63,7 +64,18 @@
   }
 
   .modal-content {
-    max-height: calc(100vh - 20px);
+    height: -webkit-fill-available;
```

Comment:
Hi @dhreb,
The CSS property `-webkit-fill-available;` has a weird behavior on mobile devices. Could you double check that it performs as expected?